src/Foundation/NSMetadataItem.cs

```diff
@@ -162,1053 +169,1347 @@ bool GetBool (NSString key)
 			}
 		}
 
-		[iOS (7,0)][Mac (10,9)]
```

Comment:
Should I keep this in? I removed it since the class has the same attributes, but it makes changes to the diff

Reply:
good catch! it's fine to remove

Reply:
Wait.. I think I just got confused. I don't think these are actually exposed by the class and do need to be included